packages/slate-sections-plugin/index.js

```diff
@@ -53,6 +56,12 @@ module.exports = class sectionsPlugin {
     );
   }
 
+  addSectionsToContext(compilation) {
+    compilation.contextDependencies.add(this.options.to);
+
+    return Promise.resolve({});
```

Comment:
Once we use the synchronous `tap()`, we can remove this.

Reply:
I just added it to the other hook we tap, as it can be done there.